uui/components/tables/columnsConfigurationModal/ColumnRow.tsx

```diff
@@ -33,11 +34,31 @@ export const ColumnRow = React.memo(function ColumnRow(props: ColumnRowProps) {
             ...(isDndAllowed ? { onTouchStart, onPointerDown } : {}),
         };
 
+        const getLabelContent = (props: ColumnsConfigurationRowProps) => {
+            return (
+                <FlexRow>
+                    <span>{props.caption}</span>
+                    { props.info && (
+                        <span style={ { fontSize: '10px', marginTop: '2px', marginLeft: '3px', color: 'gray' } }> 
```

Comment:
Why we use hire inline styles? Let's move it to the css class.

Reply:
done